services/metadactyl-clj/src/metadactyl/persistence/oauth.clj

```diff
@@ -117,10 +116,10 @@
   (let [id  (if (string? id) (UUID/fromString id) id)
         req (get-authorization-request id)]
     (when (nil? req)
-      (throw+ {:error_code ce/ERR_BAD_REQUEST
-               :reason     (str "authorization request " (str id) " not found")}))
+      (throw+ {:type  :clojure-commons.exception/not-found
```

Comment:
It seems weird that an internal authorization thing would result in a 404 -- I assume this isn't something that's called when the upstream caller is asking for the state, but when internal code is, and I think if that's a correct assumption the previous error of 400 is more correct, since whatever thing the caller is actually asking for isn't what's missing, making a 404 semantically incorrect. If I'm wrong, well, dunno :)